provisioning/roles/nodejs/templates/package.json.webpack.j2

```diff
@@ -9,25 +9,26 @@
     "build": "NODE_ENV=production webpack"
   },
   "devDependencies": {
-    "autoprefixer": "^8.2.0",
-    "babel-core": "^6.26.0",
-    "babel-loader": "^7.1.4",
-    "babel-preset-env": "^1.6.1",
-    "css-loader": "^0.28.11",
-    "cssnano": "^3.10.0",
-    "file-loader": "^1.1.11",
+    "@babel/core": "^7.1.2",
+    "@babel/preset-env": "^7.1.0",
+    "@babel/preset-flow": "^7.0.0",
+    "autoprefixer": "^9.1.5",
+    "babel-loader": "^8.0.4",
+    "css-loader": "^1.0.0",
+    "cssnano": "^4.1.4",
+    "file-loader": "^2.0.0",
     "mini-css-extract-plugin": "^0.4.0",
     "node-sass": "^4.8.3",
-    "postcss": "^6.0.21",
-    "postcss-cli": "^5.0.0",
-    "postcss-loader": "^2.1.3",
-    "sass-loader": "^6.0.7",
-    "style-loader": "^0.20.3",
-    "svg-sprite-loader": "^3.7.3",
+    "postcss": "^7.0.5",
+    "postcss-cli": "^6.0.0",
+    "postcss-loader": "^3.0.0",
+    "sass-loader": "^7.1.0",
+    "style-loader": "^0.23.1",
+    "svg-sprite-loader": "^4.1.2",
     "svgo": "^1.0.5",
     "svgo-loader": "^2.1.0",
     "webpack": "^4.5.0",
-    "webpack-cli": "^2.0.14",
+    "webpack-cli": "^3.1.2",
     "webpack-dev-server": "^3.1.3"
   },
   "browserslist": {{ webpack_browserslist | to_json }}
```

Comment:
Now that packages have been updated and we have a more recent version of browserslist, we should add `not dead` to the list defined in `webpack_browserslist` to avoid unnecessary transpilation. See https://github.com/browserslist/browserslist#full-list